.claude/CLAUDE.md

```
@@ -0,0 +1,260 @@
+`src/test/java/akuma/whiplash/common/fixture/`에 enum 기반 픽스처 존재:
+- `MemberFixture` — 테스트용 멤버 (`MEMBER_1` ~ `MEMBER_N`), `toMockEntity()` 제공
+- `AlarmFixture` — 테스트용 알람 (`ALARM_01` ~ `ALARM_N`), `toMockEntity()` 제공
+- `AlarmOccurrenceFixture`
+
+### FCM 테스트
+
+프로파일 `test`에서는 `MockFcmService`가 자동으로 등록되어 실제 FCM 요청을 보내지 않는다.
+
+## 커밋 메시지 컨벤션
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)